dev/Installer/src/install.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include "pch.h"
+#include "packages.h"
+
+struct PackageProperties
```

Comment:
This type is mixing casing models; one thing we have in the OS repo is "types are PascalCased, members are camelCased."

Reply:
Fixed to match Windows style guidelines.